frontend/pages/SoftwarePage/SoftwareAddPage/SoftwareFleetMaintained/FleetMaintainedAppDetailsPage/helpers.tsx

```diff
@@ -0,0 +1,41 @@
+import fleetAppData from "../../../../../../server/mdm/maintainedapps/apps.json";
```

Comment:
there would be a webpack build error that would fail the github action if this file was moved and this path could not be resolved.

Reply:
there would be a webpack build error that would fail the github action if this file was moved and this path could not be resolved.

as far as if the shape changed I think we can add a try catch to gracefully handle that runtime error. Or we could add a test for this file to asset the correct keys and values are present. I can look into adding that

Reply:
I will add this test on the next PR.